src/app/components/links/basic-link/basic-link.component.ts

```diff
@@ -133,6 +134,7 @@ export class BasicLinkComponent extends BaseComponent {
   @Input() public hoverColor: string = 'pink'; // Should be one of 'pink', 'black', 'gray', 'white'.
   @Input() public href: string;
   @Input() public hrefTarget: string = '_blank';
+  @Input() public rel: string = 'noopener';
```

Comment:
Yeah, you could evaluate it in an exprerssion in the html though. But this works too.